test/connectors/RegistrationConnectorTest.scala

```diff
@@ -0,0 +1,152 @@
+package connectors
+
+import base.BaseSpec
+import com.github.tomakehurst.wiremock.client.WireMock._
+import generators.Generators
+import models.`if`.IfErrorResponse
+import models.requests.RegistrationRequest
+import models.{BankDetails, ContactDetails, Registration, VatDetails}
+import org.scalacheck.Arbitrary.arbitrary
+import play.api.Application
+import play.api.http.HeaderNames._
+import play.api.http.MimeTypes
+import play.api.http.Status._
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.test.Helpers.running
+import uk.gov.hmrc.domain.Vrn
+
+import java.time.{Instant, LocalDate}
+
+class RegistrationConnectorTest extends BaseSpec with WireMockHelper  with Generators {
```

Comment:
Can we name this `RegistrationConnectorSpec` to keep it consistent across all Specs